ceph-medic.spec.in

```diff
@@ -11,40 +13,29 @@
 # common
 #################################################################################
 Name:           ceph-medic
-Version:        0.0.1
-Release:        0
+Version:        @VERSION@
+Release:        @RELEASE@%{?dist}
 Summary:        Find common issues on Ceph clusters
 License:        MIT
 Group:          System/Filesystems
-URL:            http://ceph.com/
-Source0:        %{name}-%{version}.tar.bz2
+URL:            https://github.com/ceph/ceph-medic
+Source0:        %{name}-%{version}-%{shortcommit}.tar.gz
 BuildRoot:      %{_tmppath}/%{name}-%{version}-build
 BuildRequires:  python-devel
 BuildRequires:  python-setuptools
 BuildRequires:  python-virtualenv
-BuildRequires:  python-pytest
 BuildRequires:  python-tox
-%if 0%{?suse_version}
-BuildRequires:  python-pytest
-%else
 BuildRequires:  pytest
```

Comment:
I think this might be incorrect, the package name I think is `python-pytest` not `pytest`